src/Model/Svg/SvgFile.php

```diff
@@ -616,8 +626,9 @@ public function switchToTranslationSet(array $translations): array
                     // No matching text node for this language, so we'll create one
                     $switch->appendChild($newTextTag);
                 } else {
-                    // If there is more than existing text element in a switch (with the given lang), give up on this file.
-                    throw new SvgStructureException('multiple-text-same-lang', $switch, [$language]);
+                    // If there is more than one existing text element in a switch (with the given lang), give up on this file.
```

Comment:
Do we need this else statement since this should never happen?

Reply:
Probably not — I guess I'm just being cautious by leaving it in! Do you think the tests cover this situation enough that we can be confident in removing it?

Reply:
I think I'd rather leave this in for now. There's lots here to be revisited anyway, so I think it'll be cleaned up then if need be. It'd be nice to get a new release out soon, to at least reduce the log spam.